templates/_homepage.html

```diff
@@ -1,25 +1,35 @@
+{% load emojificate %}
 {% load humanize %}
 {% load i18n %}
 {% load package_tags %}
-{% load emojificate %}
 {% load typogrify_tags %}
+{% load waffle_tags %}
 
-<div class="container">
-    <div class="row">
-        {% include "homepage/django_package_of_the_week_partial.html" %}
-        {% include "homepage/grid_of_the_week_partial.html" %}
-        {% include "homepage/public_service_announcement_partial.html" %}
+
+{% switch "switch_use_grid_of_the_week" %}
+    <div class="container">
+        <div class="row">
+            {% include "homepage/grid_of_the_week_partial.html" %}
+            {% include "homepage/django_package_of_the_week_partial.html" %}
+            {% switch "switch_use_psa" %}
```

Comment:
Does this mean we need both this and "switch_use_grid_of_the_week" switch on to make PSA visible?

Reply:
yes, exactly. We'll probably launch something for it before the PSAs. I'll update that text block to point to our mastodon since that's low-hanging fruit.

Reply:
Btw, these auto-register in the admin so there isn't a guessing game to find them.